docs/_posts/murat-gunay/2021-12-03-nerdl_fewnerd_100d_pipeline_en.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: model
+title: Pre-trained Pipeline for Few-NERD-General NER Model
+author: John Snow Labs
+name: nerdl_fewnerd_100d_pipeline
+date: 2021-12-03
+tags: [fewnerd, nerdl, en, open_source]
+task: Named Entity Recognition
+language: en
+edition: Spark NLP 3.3.1
+spark_version: 2.4
+supported: true
+article_header:
+  type: cover
+use_language_switcher: "Python-Scala-Java"
+---
+
+## Description
+
+This pretrained pipeline is built on Few-NERD model and it detects :
+
+`PERSON`, ORGANIZATION`, `LOCATION`, `ART`, `BUILDING`, `PRODUCT`, `EVENT`, `OTHER`
```

Comment:
Missing quote before ORGANIZATION

Reply:
@josejuanmartinez 
thanks for your feedback. I fixed and committed the changes. You can approve the review.